server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4468,6 +4472,16 @@ private boolean checkIfHostIsDedicated(HostVO host) {
         }
     }
 
+    private void checkIfHostOfVMIsInPrepareForMaintenanceState(Long hostId, Long vmId, String operation) {
```

Comment:
@alexandrelimassantana Added javadoc for the method "checkIfHostOfVMIsInPrepareForMaintenanceState"